src/ripple/rpc/handlers/AccountInfo.cpp

```diff
@@ -23,6 +23,7 @@
 #include <ripple/ledger/ReadView.h>
 #include <ripple/protocol/ErrorCodes.h>
 #include <ripple/protocol/Indexes.h>
+#include <ripple/protocol/RPCErr.h>
```

Comment:
Wasn't all of the `ripple` stuff renamed to `xrpl`? or is it not merged yet? I know there is an alias - i'd refactor the entire codebase to use the new `xrpl` (in a separate PR). surprised it's not already done tbh.